unreleased.rst

```diff
@@ -22,3 +22,5 @@ HEAD — ongoing
   Existing database schemas are not compatible anymore with the newer version. The extractor needs to be run on an empty schema from Ledger Begin.
 - [Sandbox] The active contract service correctly serves stakeholders. See `#2070 <https://github.com/digital-asset/daml/issues/2070>`__.
 - [Sandbox] Added the ``--maxInboundMessageSize`` CLI parameter to set the maximux size of messages received through the Ledger API. If the value is not set the current default is preserved (4 MB).
+- [DAML Compiler] Handwritten instances of ``Template`` and ``Choice`` typeclasses are no longer supported.
```

Comment:
I thought the classes were just called differently? Are they not exposed or why can’t you create instances yourself?

This would mean a giant refactoring of a codebase (you can probably guess which one :)).

Reply:
You can't create the instances now because the compiler inserts parts of the implementation based on methods in the `XInstance` (where X is a template name) typeclass.

I'll have a look at the damage to that codebase.